style.css

```diff
@@ -75,8 +77,248 @@ li {
 
 .connect {
   font-weight: 500;
+  font-size: 23px;
+  line-height: 24px;
+  color: #7f8cff;
+  padding: 1rem;
+}
+
+.col-1 {
+  align-items: center;
+}
+
+img {
+  width: 100%;
+}
+
+.tonic {
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 12px;
+  margin: 20px;
+}
+
+.greet .name {
+  padding-top: 13px;
+  padding-bottom: 13px;
+  font-size: 2.5rem;
+}
+
+.second-section {
+  display: flex;
+  flex-direction: column;
+  padding-top: 25px;
+  background-color: #f9f6f6;
+  box-shadow: -10px 10px 10px -6px rgba(207, 205, 205, 0.501);
+}
```

Comment:
- [ ] as mentioned by the previous reviewer the container of the works section which is `<section class="second-section">` should be aligned with display **_grid_** and not flex , kindly fix it 🙏

Reply:
@ichala he said flex not grid

Reply:
> @ichala he said flex not grid

@quavo19 the previous reviewer said `I think it would be better
to add display: grid in the In order to lay out the elements in the "Works" section` and the content of each project card should be aligned with flex, if you need more explanation let me know & contact me in slack 'Ali Jendoubi'

Reply:
@ichala  @TuyishimireEric there are two classes of work.  work, and work1. can you please be specific?

Reply:
@ichala  @TuyishimireEric do I display (language, framework, and skill ) as flex or ( javascript CSS and Html)  or both dives should have a display of grid?

Reply:
@quavo19 this section should be **_grid_** and not flex , that's it
![image](https://user-images.githubusercontent.com/89282221/198714330-98f7ffb3-d3d5-4547-bda5-b9c91c6da494.png)

Reply:
@ichala  @TuyishimireEric the program requirements say 

-  "In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card. " 

I used grid to contain all the cards(.second-section) and flex for the individual cards(.tonic)
but I can go ahead and change it if it pleases you.
just that I have only one review chance left